src/foremast/templates/infrastructure/app_data.json.j2

```diff
@@ -11,13 +11,37 @@
             "repoType": "github",
             "repoProjectKey": "{{ appinfo.project }}",
             "repoSlug": "{{ appinfo.repo }}",
-            "platformHealthOnlyShowOverride": true
+            "platformHealthOnlyShowOverride": true,
+            "chaosMonkey": {
+                "enabled": {{ pipeline_configs.chaos_monkey.enabled | tojson }},
+                "meanTimeBetweenKillsInWorkDays": {{ pipeline_configs.chaos_monkey.mean_time }},
+                "minTimeBetweenKillsInWorkDays": {{ pipeline_configs.chaos_monkey.min_time }},
+                "grouping": "cluster",
+                "regionsAreIndependent": true,
+                "exceptions":
+                [{% for exception in pipeline_configs.chaos_monkey.exceptions %}
+                {"account": {{ exception | tojson }}, "region": "*", "stack": "*" }
+                {% if not loop.last %},{% endif %}
+                {% endfor %}
+                {% if 'prod' not in pipeline_configs.chaos_monkey.exceptions %}
```

Comment:
Do not do this `prod*` exception in the public templates, only in our private templates. Other users probably do not have accounts named `prod`, `prods`, etc.